<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpatialDecisionDockWidgetBase</class>
 <widget class="QDockWidget" name="SpatialDecisionDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDSS Template</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="sampleWidgets">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,5,0">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="openScenarioButton">
            <property name="text">
             <string>Open Scenario</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="saveScenarioButton">
            <property name="text">
             <string>Save Scenario</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,5">
          <item row="0" column="0">
           <widget class="QLabel" name="selectLayerLabel">
            <property name="text">
             <string>Select layer:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="selectLayerCombo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="selectAttributeLabel">
            <property name="text">
             <string>Select attribute:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="selectAttributeCombo"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="medicButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/plugins/SpatialDecision/icons/medic_box.png</normaloff>:/plugins/SpatialDecision/icons/medic_box.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="ambulanceButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/plugins/SpatialDecision/icons/ambulance.png</normaloff>:/plugins/SpatialDecision/icons/ambulance.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="analysisTab">
       <attribute name="title">
        <string>analysis</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="setNetworkButton">
          <property name="text">
           <string>Build Network</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="serviceAreaButton">
          <property name="text">
           <string>ServiceArea</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="serviceAreaCutoffEdit">
          <property name="text">
           <string>1500</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="shortestRouteButton">
          <property name="text">
           <string>Shortest Route</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="networkLabel">
          <property name="text">
           <string>Uses selected objects for origins / destinations</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="bufferButton">
          <property name="text">
           <string>Buffer</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="bufferCutoffEdit">
          <property name="text">
           <string>800</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="selectBufferButton">
          <property name="text">
           <string>Select in Buffer</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="selectRangeButton">
          <property name="text">
           <string>Select in Range</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="expressionSelectButton">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLineEdit" name="expressionEdit">
          <property name="text">
           <string>id &lt; 3</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="expressionFilterButton">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="clearRouteButton">
          <property name="text">
           <string>Clear Routes</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>visualisation</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="visualisationTab">
       <attribute name="title">
        <string>reporting</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,1" rowminimumheight="0,1,1">
          <item row="0" column="0">
           <widget class="QLabel" name="featureCounterLabel">
            <property name="text">
             <string>Feature counter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="featureCounterUpdateButton">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="saveMapPathButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="saveMapButton">
            <property name="text">
             <string>Save Map</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="saveMapPathEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="featureCounterEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="reportList"/>
        </item>
        <item>
         <widget class="QTableWidget" name="statisticsTable"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
